mParticle-Rokt/MPKitRokt.m

```diff
@@ -100,10 +100,14 @@ - (MPKitExecStatus *)executeWithViewName:(NSString * _Nullable)viewName
                             filteredUser:(FilteredMParticleUser * _Nonnull)filteredUser {
     NSDictionary<NSString *, NSString *> *mpAttributes = [filteredUser.userAttributes transformValuesToString];
     NSMutableDictionary<NSString *, NSString *> *finalAtt = [[NSMutableDictionary alloc] init];
-    if (filteredUser.userId.stringValue) {
+    [finalAtt addEntriesFromDictionary:mpAttributes];
+    if (filteredUser.userId.stringValue != nil) {
         [finalAtt addEntriesFromDictionary:@{@"mpid": filteredUser.userId.stringValue}];
     }
-    [finalAtt addEntriesFromDictionary:mpAttributes];
+    NSString *sandboxKey = @"sandbox";
```

Comment:
This is obvious to us now since we're currently developing this, but we may forget in the future.  Can you add a comment here that says something like 

"The core SDK does not set sandbox on the user, but we must pass it to Rokt if provided"